hatohol.spec.in

```diff
@@ -67,55 +68,50 @@ Group: Applications/System
 Requires: hatohol-lib-arm = %{version}
 
 %description arm-zabbix
-arm-zabbix collects monitoring data from a Zabbix server and sends
+hatohol-arm-zabbix collects monitoring data from a Zabbix server and sends
 it to hatohol via HAPI.
 
 %package arm-ceilometer
-Summary: Arm for Ceilometer.
+Summary: Arm for OpenStack Ceilometer.
 Group: Applications/System
 Requires: hatohol-lib-arm = %{version}
 
 %description arm-ceilometer
-arm-ceilometer collects monitoring data from a Ceilometer and sends
-it to hatohol via HAPI.
+hatohol-arm-ceilometer collects monitoring data from a OpenStack Ceilometer
+and sends it to hatohol via HAPI.
 
 %package lib-common
 Summary: Library files for Hatohol
 Group: Applications/System
 
 %description lib-common
-Common libraries for hatohol and arm plugins.
+Common libraries for hatohol and arms.
 
 %package lib-arm
-Summary: Library files for Hatohol Arm
+Summary: Library files for Hatohol arms.
 Group: Applications/System
 Requires: hatohol-lib-common = %{version}
 
 %description lib-arm
-Libraries for arm.
+Libraries for arms.
 
 %package web
-Summary: A web UI web of Hatohol.
+Summary: Hatohol web frontend.
 Group: Applications/System
 Requires: MySQL-python >= 1.2.3
-%if %is_el6
 Requires: Django >= 1.5.3
-%endif
-%if %is_el7
-Requires: python-django
```

Comment:
I want to confirm.
We'll rebuild Django package for CentOS7, willn't we?

python-django package for CentOS7 is provided by [EPEL](http://dl.fedoraproject.org/pub/epel/7/x86_64/repoview/python-django.html).
Previously, Django package for CentOS6 isn't provided. So, We build and provide Django package for CentOS6.
(Now, it seems Django package is provided by EPEL as [python-django15](http://dl.fedoraproject.org/pub/epel/6/i386/repoview/python-django15.html).)